website/docs/d/is_lb_pool.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "VPC infrastructure"
+layout: "ibm"
+page_title: "IBM : ibm_is_lb_pool"
+description: |-
+  Get information about LoadBalancerPool
+subcategory: "Virtual Private Cloud API"
```

Comment:
which twice the subcategory please delete the subcategory: "Virtual Private Cloud API" from all the docs